workshop/views.py

```diff
@@ -143,6 +219,29 @@ def post(self, request):
         return Response(serializer.data, status=status.HTTP_200_OK)
 
 
+class EntityTagCreateView(generics.GenericAPIView):
+    permission_classes = (permissions.IsAuthenticated, AllowAnyEntityHeadOrContact,)
+    serializer_class = EntityTagCreateSerializer
+
+    def get_serializer_context(self):
+        return {
+            'request': self.request,
+            'format': self.format_kwarg,
+            'view': self,
+        }
+
+    def post(self, request):
+        """
+        Create Tag for an Entity - only Entity POR Holders or\
+        any Workshop Contact are allowed to create a tag for the entity.
+        """
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        tag = serializer.save()
+        serializer = EntityTagSerializer(tag)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+
 class TagSearchView(generics.GenericAPIView):
```

Comment:
```suggestion
class ClubTagSearchView(generics.GenericAPIView):
```
If possible, you can make such changes to make the method name consistent with that of entities, maybe in the next PR, so as to avoid any confusion in the future. What do you say? @Vikhyath08

Reply:
Again, this was because of my obsession to not modify any existing views😅. Yeah I'll make the changes.